### PR TITLE
feat: add paragraph before and after code block

### PR DESCRIPTION
**Pull request by seonim-ryu**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

## Feature

Add paragraph before and after the code block.

feature

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/editor/src/wysiwyg/nodes/codeBlock.ts [hunk 1]
```diff
@@ -46,12 +48,45 @@ export class CodeBlock extends NodeSchema {
     return () => (state, dispatch) => setBlockType(state.schema.nodes.codeBlock)(state, dispatch);
   }
 
+  moveCursor(direction: 'up' | 'down'): Command {
+    return (state, dispatch) => {
+      const { tr, doc, schema } = state;
+      const { $from } = state.selection;
+      const { view } = this.context;
```
Comment (js87zz) [inline apps/editor/src/wysiwyg/nodes/codeBlock.ts:55]: `view`는 바로 써도 될 것 같습니다~!

File: apps/editor/src/wysiwyg/nodes/codeBlock.ts [hunk 2]
```diff
@@ -46,12 +48,45 @@ export class CodeBlock extends NodeSchema {
...
+
+      if (view!.endOfTextblock(direction) && $from.node().type.name === 'codeBlock') {
+        const lines = $from.parent.textContent.split('\n');
+
+        const offset = direction === 'up' ? $from.start() : $from.end();
+        const range =
+          direction === 'up'
+            ? [offset, lines[0].length + offset]
+            : [offset - lines[lines.length - 1].length, offset];
```
Comment (js87zz) [inline apps/editor/src/wysiwyg/nodes/codeBlock.ts:64]: 유틸에 `last` 함수 사용해도 좋을 것 같습니다!

File: apps/editor/src/wysiwyg/nodes/codeBlock.ts [hunk 3]
```diff
@@ -46,12 +48,45 @@ export class CodeBlock extends NodeSchema {
...
+        const pos = doc.resolve(direction === 'up' ? $from.before() : $from.after());
```
Comment (js87zz) [inline apps/editor/src/wysiwyg/nodes/codeBlock.ts:65]: `resolve`메서드를 사용해서 `resolvedPos`를 넘길 필요는 없어보입니다. 바로 찾은 위치를 넘겨줘도 될 것 같아요~!

**Review by js87zz — APPROVED** [review]
리뷰 완료입니다. 커스텀 블럭도 동작을 한번 확인해봐야겠군요

File: apps/editor/src/wysiwyg/nodes/codeBlock.ts (hunk 3 above)
Comment (seonim-ryu) [reply apps/editor/src/wysiwyg/nodes/codeBlock.ts:65]: > 66번 라인에서 사용하기 위해서 `resolve` 메소드 호출함

**Pull request merged**